Simplenote/SPAppDelegate.m

```diff
@@ -195,6 +195,7 @@ - (void)applicationDidEnterBackground:(UIApplication *)application
 - (void)applicationWillEnterForeground:(UIApplication *)application
 {
     [self dismissPasscodeLockIfPossible];
+    [self.simperium authenticateIfNecessary];
```

Comment:
Wonder if there is a better way 🤔 
I think this method works because it forces web socket reconnection.

Reply:
YES we should avoid doing this. Ongoing conversation via P2, outcome still TBD!

Reply:
Per the continuing conversation I have added a timestamp stored in app delegate to mark a deletion request.  Only if a deletion request was made in the last 24 hours will the [self.simperium authenticateIfNecessary]; get called